### PR TITLE
Made the front end using bootstrap 4

### LINKED ISSUE #3: Make the front-end for the Flask app
The front-end should a text box for the input of the username, a button for running the script. Make the front-end for displaying the results, once the #2 is done.

### PR DESCRIPTION
**Pull request by adiaux**
Developed the front end using bootstrap 4 and added some animations. Fixes #3. @vchrombie You can also add some new images if needed. If you have any ideas, please suggest.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "chtenv/bin/python3.6"
+}
```
Comment (vchrombie) [inline .vscode/settings.json]: What is this?

File: app.py [hunk 2]
```diff
@@ -55,13 +55,12 @@ def getpullRequests(username):
...
         if prsdata:
-            #print(prsdata)
+            print(prsdata)
             return render_template('prs.html', prs=prsdata)
         else:
             print("No PRs made in Hacktoberfest")
             return render_template('prs.html')
 
 if __name__ == "__main__":
-	app.debug = True
```
Comment (vchrombie) [inline app.py]: Why did you remove this?

File: app.py [hunk 3]
```diff
@@ -55,13 +55,12 @@ def getpullRequests(username):
                 prdata['deletions'] = pr['deletions']
                 prsdata[pr['id']] = prdata
         if prsdata:
-            #print(prsdata)
+            print(prsdata)
```
Comment (vchrombie) [inline app.py]: ?

File: scripts/script.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 import collections
 import re 
 
-accessToken = "<--insert your accessToken here-->"
+accessToken = "96e0acd5aba4e96d15757024575f570a9bfa2dcb"
```
Comment (vchrombie) [inline scripts/script.py]: :man_shrugging:

File: secret.py [hunk 5]
```diff
@@ -1 +1 @@
-accessToken = "<<insert your github personal access token here>>"
\ No newline at end of file
+accessToken = "96e0acd5aba4e96d15757024575f570a9bfa2dcb"
```
Comment (vchrombie) [inline secret.py]: :man_shrugging: :man_facepalming:

**Comment by vchrombie** [comment]
Can you remove the bootstrap files and maybe use an online CDN for bootstrap?

**Comment by vchrombie** [comment]
> ## Developed the front end using bootstrap 4 and added some animations. Fixes #3 . @vchrombie You can also add some new images if needed.If you have any ideas, please suggest

And this is wrong. Headings are meant to be different apart from the normal paragraphs. Please edit this.

File: .vscode/settings.json (hunk 1 above)
Comment (SkaarFacee) [reply .vscode/settings.json]: I guess its an extension for vscode, will get fix that :)

**Comment by vchrombie** [comment]
Still, you have use CDN instead of adding a few more files. Maybe you can use [cdnjs.com](cdnjs.com) to handle all such imports.

File: templates/index.html [hunk 6]
```diff
@@ -1,24 +1,72 @@
...
+			</div>
 		</div>
+		</form>
+	
+		
+		
+		
 
+		<script src="js/jquery-3.3.1.min.js"></script>
+		<script src="bootstrap-4.1.3-dist/js/bootstrap.min.js"></script>
```
Comment (vchrombie) [inline templates/index.html]: Is this file existing?

File: templates/index.html [hunk 7]
```diff
@@ -1,24 +1,72 @@
...
+				</div>	
+			</div>
 		</div>
+		</form>
+	
+		
+		
+		
 
+		<script src="js/jquery-3.3.1.min.js"></script>
```
Comment (vchrombie) [inline templates/index.html]: You can use the CDNs from [cdnjs.com](cdnjs.com).

File: templates/prs.html [hunk 8]
```diff
@@ -2,33 +2,82 @@
...
-        </table>
-        {% else %}
-            <h3 class="text-center">No PRs made in Hacktoberfest</h3>
-        {% endif %}
-    </div>
+   		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+		<link rel="stylesheet" href="css/animate.css">
```
Comment (vchrombie) [inline templates/prs.html]: The same applies here. Use [cdnjs.com](cdnjs.com) wherever possible.

**Comment by vchrombie** [comment]
@adiaux 
There are still a few alien files in the PR. Take time and finish it. Make sure your PR has the correct files which are supposed to be there.

**Comment by vchrombie** [comment]
Add commits to the same PR. You can squash the commits in the last after I approve.

**Comment by adiaux** [comment]
Sent a new PR

File: scripts/script.py [hunk 9]
```diff
@@ -3,7 +3,7 @@
 import collections
 import re 
 
-accessToken = "<--insert your accessToken here-->"
+accessToken = "99b3a7d957abe18da9be9aca3fcda6189d9c00b7"
```
Comment (vchrombie) [inline scripts/script.py]: again! @adiaux :man_shrugging:

File: secret.py [hunk 10]
```diff
@@ -1 +1 @@
-accessToken = "<<insert your github personal access token here>>"
\ No newline at end of file
+accessToken = "99b3a7d957abe18da9be9aca3fcda6189d9c00b7"
```
Comment (vchrombie) [inline secret.py]: check once before committing :man_shrugging:

**Comment by vchrombie** [comment]
Please resolve all the comments when you solve them and are expired.

**Comment by vchrombie** [comment]
> Add commits to the same PR. You can squash the commits in the last after I approve.

Also, remember this.

File: .vscode/settings.json (hunk 1 above)
Comment (SkaarFacee) [reply .vscode/settings.json]: Got that fixed :)

File: app.py (hunk 3 above)
Comment (SkaarFacee) [reply app.py]: Used it for debugging purpose, I've made it back to normal

File: scripts/script.py (hunk 4 above)
Comment (SkaarFacee) [reply scripts/script.py]: My bad

File: secret.py (hunk 5 above)
Comment (SkaarFacee) [reply secret.py]: My bad again

**Comment by vchrombie** [comment]
And one more thing, you fixed the query issue thing in this PR. Make sure you address the different issues in different PRs.

File: templates/index.html (hunk 6 above)
Comment (SkaarFacee) [reply templates/index.html]: Removed them, they were not is use

File: templates/index.html (hunk 7 above)
Comment (SkaarFacee) [reply templates/index.html]: Done :+1:

File: templates/prs.html (hunk 8 above)
Comment (SkaarFacee) [reply templates/prs.html]: Done

File: scripts/script.py (hunk 9 above)
Comment (SkaarFacee) [reply scripts/script.py]: Umm :thinking:

File: secret.py (hunk 10 above)
Comment (SkaarFacee) [reply secret.py]: Okay :+1:

**Comment by adiaux** [comment]
Query issue still persists, working on it right now

**Comment by vchrombie** [comment]
> Query issue still persists, working on it right now

Better you send a new PR for that. Maybe create a new branch from the master and fix the problem in that branch. Let this PR deal with only the front-end.

**Comment by vchrombie** [comment]
LGTM, merging it right away. Thanks for the PR, @adiaux. :tada:
